src/@utils/aquarius/index.ts

```diff
@@ -56,6 +68,39 @@ export function getFilterTerm(
   }
 }
 
+export function getFilter(...args: any[]) {
+  let filters = []
+  if (typeof args[0] === 'object') {
+    args[0].forEach((arg) => {
+      const filter = arg.split('=')
+      filters = [...filters, filter]
+    })
+  } else {
+    const filter = args[0].split('=')
+    filters = [...filters, filter]
+  }
+
+  let filter: Query[] = []
+  filters.forEach((filterItem) => {
+    let query: Query = {
+      must: {
+        exists: { field: filterItem[0] }
+      }
+    }
+    if (filterItem[1] === 'MUST_EXISTS_AND_NON_EMPTY') {
```

Comment:
suggestion(blocking): Move FILTER_VALUES const into this file and reference it to prevent errors: `FILTER_VALUES.MUST_EXISTS_AND_NON_EMPTY`